config/jobs/kubernetes-sigs/sig-windows/release-1.23-windows-presubmits.yaml

```diff
@@ -129,7 +129,7 @@ presubmits:
     extra_refs:
     - org: kubernetes-sigs
       repo: azuredisk-csi-driver
-      base_ref: master
+      base_ref: release-1.9
```

Comment:
I think we want to keep this ref at master.
@jsturtevant @andyzhangx FYI

Reply:
I would leave it as is